### PR TITLE
Google-drive/get parents reco

### PR DESCRIPTION
**Pull request by GalNakash-RecoLabs**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive.yml [hunk 1]
```diff
@@ -3076,7 +3076,32 @@ script:
     - contextPath: GoogleDrive.File.File.mimeType
       description: The MIME type of the copied file.
       type: String
-  dockerimage: demisto/googleapi-python3:1.0.0.83610
+  - name: google-drive-file-get-parents
```
Comment (YuvHayun) [inline Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive.yml:3079]:
```suggestion
  - name: google-drive-get-file-parents
```

Comment (YuvHayun) [inline Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive.yml:3079]: Also, make sure to add this info to the readme file.

File: Packs/GoogleDrive/ReleaseNotes/1_3_0.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Google Drive
+
+- Get list of parents directory command
```
Comment (YuvHayun) [inline Packs/GoogleDrive/ReleaseNotes/1_3_0.md:6]:
```suggestion
- Added the **google-drive-get-file-parents** command.
```

File: Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive_test.py [hunk 3]
```diff
@@ -1084,3 +1084,34 @@ def raise_error():
...
+    @patch(MOCKER_HTTP_METHOD)
+    def test_drive_get_file_get_parents_success(self, mocker_http_request, gsuite_client):
+        """
+        Scenario: For file_get_parents command successful run.
+
+        Given:
+        - Command args.
+
+        When:
+        - Calling google-drive-drive-get command with the parameters provided.
```
Comment (YuvHayun) [inline Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive_test.py:1097]:
```suggestion
        - Calling google-drive-get-file-parents command with the parameters provided.
```

File: Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive_test.py [hunk 4]
```diff
@@ -1084,3 +1084,34 @@ def raise_error():
                     'user_id': 'test_user_id',
                 }
             )
+
+    @patch(MOCKER_HTTP_METHOD)
+    def test_drive_get_file_get_parents_success(self, mocker_http_request, gsuite_client):
```
Comment (YuvHayun) [inline Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive_test.py:1089]:
```suggestion
    def test_drive_get_file_parents_success(self, mocker_http_request, gsuite_client):
```

File: Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive.py [hunk 5]
```diff
@@ -1056,6 +1058,60 @@ def file_get_command(client: 'GSuiteClient', args: dict[str, str]) -> CommandRes
     return handle_response_single_file(response, args)
 
 
+@logger
+def file_get_parents(client: 'GSuiteClient', args: dict[str, str]) -> CommandResults:
+    """
+    google-drive-file-get
+    Query a single file in Google Drive.
```
Comment (YuvHayun) [inline Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive.py:1065]:
```suggestion
    google-drive-get-file-parents
    Query a single file in Google Drive to retrieve its parents.
```

File: Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive.py [hunk 6]
```diff
@@ -1056,6 +1058,60 @@ def file_get_command(client: 'GSuiteClient', args: dict[str, str]) -> CommandRes
...
+    :param client: Client object.
+    :param args: Command arguments.
+
+    :return: Command Result.
+    """
+
+    return file_get_parents_recursive(client, args)
+
+
+def file_get_parents_recursive(client: 'GSuiteClient', args: dict[str, str]) -> CommandResults:
```
Comment (YuvHayun) [inline Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive.py:1076]: let's put this logs under the main function.

**Review by YuvHayun — CHANGES_REQUESTED** [review]
Hey @GalNakash-RecoLabs, thx a lot for your contribution!
I've added several small comments.
Also:
- Please try to [squash commits](https://stackoverflow.com/questions/5189560/how-do-i-squash-my-last-n-commits-together) to fix the form issue.
- Make sure to run the `demisto-sdk pre-commit` command to fix the failures.
Thx again and let met know if any question comes up.

**Comment by GalNakash-RecoLabs** [comment]
@YuvHayun fixed!

**Pull request closed without merging**